packages/trpc/server/middlewares/sessionMiddleware.ts

```diff
@@ -96,10 +103,17 @@ export async function getUserFromSession(ctx: TRPCContextInner, session: Maybe<S
   // This helps to prevent reaching the 4MB payload limit by avoiding base64 and instead passing the avatar url
   user.avatar = rawAvatar ? `${WEBAPP_URL}/${user.username}/avatar.png` : defaultAvatarSrc({ email });
   const locale = user?.locale || ctx.locale;
+
+  const isOrgAdmin = !!user.organization?.members.length;
+  // Want to reduce the amount of data being sent
+  if (isOrgAdmin && user.organization?.members) {
+    user.organization.members = [];
+  }
```

Comment:
To reduce the payload being sent to the front we don't have to return the members array. It's just the user's information so it would have been duplicated.

Reply:
Can we delete instead of sending an empty array? This might cause unintended expectations